media-libs/libkate/libkate-0.4.1-r3.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit ltprune python-single-r1
+
+DESCRIPTION="Codec for karaoke and text encapsulation for Ogg"
+HOMEPAGE="https://code.google.com/p/libkate/"
+SRC_URI="https://libkate.googlecode.com/files/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+
+IUSE="debug doc wxwidgets"
+REQUIRED_USE="wxwidgets? ( ${PYTHON_REQUIRED_USE} )"
+
+COMMON_DEPEND="
+	media-libs/libogg:=
+	media-libs/libpng:0=
+"
+DEPEND="${COMMON_DEPEND}
+	virtual/pkgconfig
+	sys-devel/flex
+	sys-devel/bison
+	doc? ( app-doc/doxygen )
+"
+RDEPEND="${COMMON_DEPEND}
+	wxwidgets? (
+		${PYTHON_DEPS}
+		dev-python/wxpython:3.0[${PYTHON_USEDEP}]
+		media-libs/liboggz )
+"
+
+pkg_setup() {
+	use wxwidgets && python-single-r1_pkg_setup
+}
+
+src_configure() {
+	econf \
+		--disable-static \
```

Comment:
For what earthly purpose? If nobody ever cared to request static libs for this package, I don't see any reason to introduce this useless burden which will turn into PITA once another upstream decides to go for half-broken CMake files that can't do static+shared.